frontend/src/bundles/admin/verification/pages/styles.module.scss

```diff
@@ -0,0 +1,178 @@
+:root {
+    --page-title-height: 82px;
+    --page-vertical-padding: 70px;
+    --content-padding-mobile: 0px 15px 0px;
+    --content-padding-web: 20px 50px 50px 50px;
```

Comment:
I think it`s better to remove bottom padding, because on my screen it adds unnecessary scrolling
![image](https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/assets/67747035/dff73102-bacb-4cc4-91a3-4ff2bcfa8236)

Reply:
Good catch